subprojects/docs/src/docs/userguide/running-builds/introduction/dependency_management_basics.adoc

```diff
@@ -0,0 +1,253 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dependency_management_basics]]
+= Dependency Management Basics
+
+Dependency management is an automated technique for declaring, resolving, and using functionality required by a project.
+
+Gradle has built-in support for dependency management.
+
+image::gradle-basic-7.png[]
+
+Gradle build scripts defines a process to build projects that contain some dependencies.
+Dependencies refer to JARs, plugins, libraries, or source code that support building your project.
+
+The Gradle build system makes it easy to include external binaries or other library modules to your build as dependencies.
+The dependencies can be located on your machine or in a remote repository, and any transitive dependencies they declare are also automatically included.
+
+== Viewing Project Dependencies
+
+You can view your dependency tree in the terminal using the `./gradlew :app:dependencies` command:
+
+[source]
+----
+$ ./gradlew :app:dependencies
+
+> Task :app:dependencies
+
+------------------------------------------------------------
+Project ':app'
+------------------------------------------------------------
+
+compileClasspath - Compile classpath for source set 'main'.
+\--- com.google.guava:guava:32.1.2-jre
+     +--- com.google.guava:failureaccess:1.0.1
+     +--- com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava
+     +--- com.google.code.findbugs:jsr305:3.0.2
+     +--- org.checkerframework:checker-qual:3.12.0
+     +--- com.google.errorprone:error_prone_annotations:2.11.0
+     \--- com.google.j2objc:j2objc-annotations:1.3
+
+...
+----
+
+== Configurations
+
+Dependencies in Gradle are grouped into sets called *configurations*.
+
+TIP: A *configuration* is simply a named set of dependencies.
```

Comment:
```suggestion
TIP: For the purposes of this introduction, you can think of a *configuration* as simply a named set of dependencies.
```

❗ This is a dramatic oversimplification that definitely needs to be qualified.  People get very confused about advanced usage of configurations (justifiably, as it is confusing).

Reply:
We decided not to describe configurations in details and just mention `implementation` and `testImplementation` in the Running guide